rpc/utils.go

```diff
@@ -95,3 +98,22 @@ func jsonify(i interface{}) string {
 	}
 	return string(jsonb)
 }
+
+func GetBlockIDAsString(blockID interface{}) string {
+	switch v := blockID.(type) {
+	case tezos.BlockHash:
+		return blockID.(tezos.BlockHash).String()
+	case int64:
+		return strconv.FormatInt(blockID.(int64), 10)
+	case string:
+		if blockID.(string) == "head" {
+			return "head"
+		} else if blockID.(string) == "genesis" {
+			return "genesis"
+		} else {
+			return fmt.Sprintf("Block tag %s not supported.\n", v)
```

Comment:
Any error case here should return an `error` type which you can create with e.g. `fmt.Errorf()`.